smoke-tests/framework/src/main/java/com/microsoft/applicationinsights/smoketest/SmokeTestExtension.java

```diff
@@ -242,11 +249,17 @@ public void beforeEach(ExtensionContext context) throws Exception {
 
   protected String getBaseUrl() {
     String appContext = getAppContext();
-    if (appContext.isEmpty()) {
-      return "http://localhost:" + appServerPort;
-    } else {
-      return "http://localhost:" + appServerPort + "/" + appContext;
+    StringBuilder sb = new StringBuilder();
+    sb.append("http://localhost");
+    if (appServerPort != 80) {
```

Comment:
i see, you only append port when it's not 80 (custom port). can we do that opposite instead... default, we don't append, but when it's 80, append?